### DISCUSSION TITLE
How can I save my entries?

### DISCUSSION THREAD

**Discussion by REDLIGHTFILMS** [react-hook-form/react-hook-form#2041] [Q&A]
I do not understand how I can save my data. How do I use "FormProvider" in this case. Do I still have to create a global state?

https://codesandbox.io/s/problem-1cxx9

**Comment by bluebill1049** [comment]
what do you mean save data? save to you local state or your API end point?

**Reply by REDLIGHTFILMS** [reply]
I want to save my data locally, so that the text fields are filled in when I jump back and forth. The data should be taken from the local state at the end of the form and sent to an API endpoint.

**Reply by bluebill1049** [reply]
then you can save into state lib, eg: redux

**Reply by REDLIGHTFILMS** [reply]
But how can I do it without a state library?

**Reply by bluebill1049** [reply]
then just use `useState`
